Error:Cause: error=2, No such file or directory

Hi,

I downloaded linphone-android project using this command "git clone git://git.linphone.org/linphone-android.git --recursive". Then use these commands "./prepare.py" and then "make". It successfully build whole project. But when I open it in android studio, I face this error "Error:Cause: error=2, No such file or directory" at bottom messages tab. In terminal using make command project build successful. But In android-studio, it doesn't build. I used android-studio 2.3.2 version. Is there I missing or doing some thing wrong?

This is the terminal log after use ./gradlew command


ronit@ronit-HP-EliteBook-8460p:~/Desktop/linphone-android$ ./gradlew build
[Push Notification] Firebase disabled
error: no devices/emulators found
:preBuild UP-TO-DATE
:preDebugBuild UP-TO-DATE
:checkDebugManifest
:preDebugAndroidTestBuild UP-TO-DATE
:preReleaseBuild UP-TO-DATE
:prepareComAndroidSupportMultidex101Library
:prepareComAndroidSupportSupportCompat2600Alpha1Library
:prepareComAndroidSupportSupportCoreUi2600Alpha1Library
:prepareComAndroidSupportSupportCoreUtils2600Alpha1Library
:prepareComAndroidSupportSupportFragment2600Alpha1Library
:prepareComAndroidSupportSupportMediaCompat2600Alpha1Library
:prepareComAndroidSupportSupportV42600Alpha1Library
:prepareDebugDependencies
:compileDebugAidl UP-TO-DATE
:compileDebugRenderscript UP-TO-DATE
:generateDebugBuildConfig UP-TO-DATE
:generateDebugResValues UP-TO-DATE
:generateDebugResources UP-TO-DATE
:mergeDebugResources UP-TO-DATE
:processDebugManifest UP-TO-DATE
:processDebugResources UP-TO-DATE
:generateDebugSources UP-TO-DATE
:incrementalDebugJavaCompilationSafeguard UP-TO-DATE
:javaPreCompileDebug
:compileDebugJavaWithJavac UP-TO-DATE
:compileDebugNdk NO-SOURCE
:compileDebugSources UP-TO-DATE
:mergeDebugShaders UP-TO-DATE
:compileDebugShaders UP-TO-DATE
:generateDebugAssets UP-TO-DATE
:mergeDebugAssets UP-TO-DATE
:transformClassesWithJarMergingForDebug UP-TO-DATE
:transformClassesWithMultidexlistForDebug UP-TO-DATE
:transformClassesWithDexForDebug UP-TO-DATE
:mergeDebugJniLibFolders UP-TO-DATE
:transformNativeLibsWithMergeJniLibsForDebug UP-TO-DATE
:transformNativeLibsWithStripDebugSymbolForDebug UP-TO-DATE
:processDebugJavaRes UP-TO-DATE
:transformResourcesWithMergeJavaResForDebug UP-TO-DATE
:validateSigningDebug
:packageDebug UP-TO-DATE
:assembleDebug UP-TO-DATE
:checkReleaseManifest
:prepareReleaseDependencies
:compileReleaseAidl UP-TO-DATE
:compileReleaseRenderscript UP-TO-DATE
:generateReleaseBuildConfig UP-TO-DATE
:generateReleaseResValues UP-TO-DATE
:generateReleaseResources UP-TO-DATE
:mergeReleaseResources UP-TO-DATE
:processReleaseManifest UP-TO-DATE
:processReleaseResources UP-TO-DATE
:generateReleaseSources UP-TO-DATE
:incrementalReleaseJavaCompilationSafeguard UP-TO-DATE
:javaPreCompileRelease
:compileReleaseJavaWithJavac UP-TO-DATE
:compileReleaseNdk NO-SOURCE
:compileReleaseSources UP-TO-DATE
:mergeReleaseShaders UP-TO-DATE
:compileReleaseShaders UP-TO-DATE
:generateReleaseAssets UP-TO-DATE
:mergeReleaseAssets UP-TO-DATE
:transformClassesWithJarMergingForRelease UP-TO-DATE
:transformClassesWithMultidexlistForRelease UP-TO-DATE
:transformClassesWithDexForRelease UP-TO-DATE
:mergeReleaseJniLibFolders UP-TO-DATE
:transformNativeLibsWithMergeJniLibsForRelease UP-TO-DATE
:transformNativeLibsWithStripDebugSymbolForRelease UP-TO-DATE
:processReleaseJavaRes UP-TO-DATE
:transformResourcesWithMergeJavaResForRelease UP-TO-DATE
:validateSigningRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':validateSigningRelease'.
> Keystore file /home/ronit/Desktop/linphone-android/"" not found for signing config 'release'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.747 secs

--- a/build.gradle
+++ b/build.gradle
@@ -184,7 +184,7 @@ android.applicationVariants.all { variant ->
 
 task runApplication() {
   def result = exec {
-    executable = 'adb'
+    executable = android.getAdbExe().toString()
     ignoreExitValue true
     args = ['shell', 'monkey', '-p', getPackageName(), '-c', 'android.intent.category.LAUNCHER', '1']
   }

